lib/alphabetical-list-items.js

```diff
@@ -22,9 +20,7 @@ function alphaCheck(ast, file, language, done) {
 
     items.forEach((item) => {
       if (item.children.length) {
-        const lineStartOffset = item.children[0].children[0].position.start.offset;
-        const lineEndOffset = item.children[0].children[item.children[0].children.length - 1].position.end.offset;
-        const text = normalize(remark.process(contents.slice(lineStartOffset, lineEndOffset)));
+        const text = normalize(toString(item));
```

Comment:
Take a look at the following Markdown list:

```
* a
  * z
* a a
```

I consider it sorted. If we "toString" the whole `a` node, `a z` will be compared to `a a`, and this linter plugin will suggest to swap `a` with `a a`. :)